src/Plugins/LevelDBStore/Plugins/Storage/Store.cs

```diff
@@ -44,7 +44,7 @@ public ISnapshot GetSnapshot()
             return new Snapshot(db);
         }
 
-        public void Put(byte[] key, byte[] value)
+        public virtual void Put(byte[] key, byte[] value)
```

Comment:
This should not be `virtual`, my reasoning its an `Interface` method and if you want to inherit class `Store` you can better options` like `new`, i doubt someone you `override` and call `base` method.

Reply:
Not really sure how this was posted....